webapp/api/dashboard.py

```diff
@@ -71,6 +71,17 @@
 ])
 
 
+DASHBOARD_API_V2 = os.getenv(
+    'SNAPCRAFT_IO_API_V2',
+    'https://dashboard.snapcraft.io/api/v2/'
+)
+
+SNAP_RELEASE_HISTORY_URL = ''.join([
+    DASHBOARD_API_V2,
+    'snaps/{snap_name}/releases?page=1&size=100'
```

Comment:
By default page size is 500, but it seems that it results in time out for some bigger snaps (node, android-studio). Reducing size to 100 seems to help for some, but others still time out (node).

I'm not sure if it's an issue on our side (should we wait longer) or on store side.

Reply:
I think it's probably our side. We're pretty tight on the timeouts